pyflakes/checker.py

```diff
@@ -1335,3 +1335,19 @@ def EXCEPTHANDLER(self, node):
                 del self.scope[node.name]
             except KeyError:
                 pass
+
+    def ANNASSIGN(self, node):
+        """
+        Annotated assignments don't have annotations evaluated on function
+        scope, hence the custom implementation.
+
+        See: PEP 526.
+        """
+        if node.value:
+            # If the node has no value, it's not really ast.Store and shouldn't
+            # silence UndefinedLocal warnings.
+            self.handleNode(node.target, node)
+        if not isinstance(self.scope, FunctionScope):
+            self.handleNode(node.annotation, node)
+        if node.value:
```

Comment:
FINE.

Agreed, it's little code but has an alarming density of overly generic terms (you're saying I should HANDLE the NODE which is the parent node's TARGET? Alrighty, and now let's HANDLE the NODE which is the parent node's VALUE!) 🤣